package.json

```diff
@@ -15,22 +15,24 @@
   },
   "main": "lib/kv-redis.js",
   "scripts": {
-    "test": "mocha test/integration/*.js",
+    "test": "mocha test/unit/*.js && mocha test/integration/*.js",
```

Comment:
I suspect this breaks the build-framework test runner. Can this not be run with a single `mocha` command?